src/performance/connection.ts

```diff
@@ -0,0 +1,31 @@
+import { BooleanFlag, Event } from "@clarity-types/data";
+import { ConnectionData, NavigatorConnection } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let data: ConnectionData;
+
+export function start(): void {
+    if (navigator && "connection" in navigator) {
+        (navigator["connection"] as NavigatorConnection).addEventListener("change", recompute);
+        recompute();
+    }
+}
+
+function recompute(): void {
+    let connection = navigator["connection"] as NavigatorConnection;
+    data = {
+        downlink: connection.downlink,
```

Comment:
Add downlinkMax,, type

Reply:
Didn't see it being supported by Chrome. Didn't want to add a null field unless we have a use case for it.